src/lib/ggpo/types.h

```diff
@@ -40,16 +40,16 @@ typedef char int8;
 typedef short int16;
 typedef int int32;
 
+
 /*
  * Macros
  */
 #if defined(_DEBUG)
-#define BREAK  DebugBreak();
+#  define BREAK  DebugBreak();
 #else
-#define BREAK  exit(1);
```

Comment:
If we're not in debug builds, we probably want to just keep going.  The SDK shouldn't ever `exit`, in any case (that should be a decision left up to the application.

Reply:
See above.